### PR TITLE
Datasets

### PR DESCRIPTION
**Pull request by ahasverus**
New features:
- Replace biodiversity datasets
- Rename these datasets names (i.e. `species_occs` > `site_species`)
- Document new datasets
- Replace climate rasters (raw data)
- Adapt examples, tests, and vignette with new datasets names

### CODE REVIEW

File: R/datasets.R [hunk 1]
```diff
@@ -1,58 +1,101 @@
-#' Coordinates of Pennsylvania survey sites
+#' Data: sites x locations object
```
Comment (Rekyt) [inline R/datasets.R:1]: Maybe add the dataset detail in the title like:
`Data: sites x locations of Western Europe mammals occurrence`

File: R/datasets.R [hunk 2]
```diff
@@ -1,58 +1,101 @@
...
+#' class(site_locations)
+#' dim(site_locations)
+#' head(site_locations)
 
+"site_locations"
 
 
-#' Tree species counts in Pennsylvania survey sites
+
+#' Data: sites x species object
```
Comment (Rekyt) [inline R/datasets.R:32]: Same remark as above for the sake of clarity ;)

File: R/datasets.R [hunk 3]
```diff
@@ -1,58 +1,101 @@
...
+#' dim(site_species)
+#' site_species[1:6, 1:6]
 
-"species_occs"
+"site_species"
 
 
 
-#' Tree species traits
+#' Data: species x traits object
```
Comment (Rekyt) [inline R/datasets.R:65]: Again!

File: R/datasets.R [hunk 4]
```diff
@@ -1,58 +1,101 @@
...
-#'   \item{n_mass}{Mass}
-#'   \item{wood_dens}{Wood density}
+#'   \item{adult_body_mass}{adult body mass (in grams)}
+#'   \item{gestation_length}{length of time of non-inactive fetal growth 
+#'   (in days)}
+#'   \item{litter_size}{number of offspring born per litter per female}
+#'   \item{max_longevity}{maximum adult age (in months)}
+#'   \item{sexual_maturity_age}{age when individuals are first physically 
+#'   capable of reproducing (in days)}
+#'   \item{diet_breadth}{number of dietary categories eaten}
```
Comment (Rekyt) [inline R/datasets.R:91]: For `diet_breadth`, maybe just "number of types of dietary categories"?

File: R/fb_aggregate_site_data.R [hunk 5]
```diff
@@ -27,15 +27,15 @@
...
+#' data("site_locations")
+#' data("site_species")
 #' 
 #' ## Import grid ----
 #' tavg <- system.file("extdata", "annual_mean_temp.tif", package = "funbiogeo")
 #' tavg <- terra::rast(tavg)
 #' 
 #' ## Rasterize 3 first species counts ----
-# FIXME (finish examples)
+# F# IXME (finish examples)
```
Comment (Rekyt) [inline R/fb_aggregate_site_data.R:38]: Is there a reason for the change in FIXME?

File: R/datasets.R (hunk 1 above)
Comment (ahasverus) [reply R/datasets.R:1]: Ok, I will do it (as for other datasets).

File: R/datasets.R (hunk 4 above)
Comment (ahasverus) [reply R/datasets.R:91]: Ok, I will do it.

File: R/fb_aggregate_site_data.R (hunk 5 above)
Comment (ahasverus) [reply R/fb_aggregate_site_data.R:38]: Nope. I will add the example now I have added new raster data.

**Pull request merged**